rattle.cabal

```diff
@@ -52,7 +52,11 @@ library
         unordered-containers >= 0.2.7,
         js-dgtable,
         js-flot,
-        js-jquery
+        js-jquery,
+        cereal >= 0.5.8.1,
+        utf8-string >= 1.0.1.1,
+        binary >= 0.8.6.0,
+        unix >= 2.7.2.2
```

Comment:
This can't be an unconditional dependency, see what I do in Shake - https://github.com/ndmitchell/shake/blob/master/shake.cabal#L120-L121. This library isn't available on Windows